src/components/AppCard.tsx

```diff
@@ -1,23 +1,133 @@
-import React from 'react';
+import React, { useState } from 'react';
+// import style from './AppCardEdit.module.css';
+import binIcon from '../images/Kind=Delete.svg';
 import styles from './AppCard.module.css';
 import editIcon from '../images/Kind=Edit.svg';
 
+interface Flashcard {
+	front: string;
+	back: string;
+}
+
 export const AppCard = () => {
-	const textArr: Array<string> = [
-		'What is Lorem Ipsum?',
-		'Lorem Ipsum is simply dummy text of the printing',
-		'and typesetting industry. Lorem Ipsum has been the industry',
-		'standard dummy text ever since the 1500s, when an unknown printer',
-		'took a galley of type and scrambled it to make a type',
-		'specimen book. It has survived not only five centuries, but also the leap',
-		'into electronic typesetting, remaining essentially unchanged.',
+	const textArr: Flashcard[] = [
+		{ front: 'What is Lorem Ipsum?', back: 'fish' },
+		{ front: 'Lorem Ipsum is simply dummy text of the printing', back: 'fish' },
+		{
+			front: 'and typesetting industry. Lorem Ipsum has been the industry',
+			back: 'fish',
+		},
+		{
+			front:
+				'standard dummy text ever since the 1500s, when an unknown printer',
+			back: 'fish',
+		},
+		{
+			front: 'took a galley of type and scrambled it to make a type',
+			back: 'fish',
+		},
+		{
+			front:
+				'specimen book. It has survived not only five centuries, but also the leap',
+			back: 'fish',
+		},
+		{
+			front: 'into electronic typesetting, remaining essentially unchanged.',
+			back: 'fish',
+		},
 	];
-	return textArr.map((item, index) => (
-		<div className={styles.card} key={index}>
-			<div className={styles.editConteiner}>
-				<img src={editIcon} alt='edit' />
-			</div>
-			<h4 className={styles.text}>{item}</h4>
-		</div>
-	));
+
+	const [cards, setCards] = useState<Flashcard[]>(textArr);
+	const [isEditing, setIsEditing] = useState<boolean[]>(
+		Array(textArr.length).fill(false)
+	);
+	const [editMode, setEditMode] = useState<boolean>(true);
+	const [secondPageVisable, setSecondPageVisable] = useState<boolean>(true);
+	const toggleCardSide = (index: number) => {
+		const newEditingState = [...isEditing];
+		newEditingState[index] = !newEditingState[index];
+		setIsEditing(newEditingState);
+	};
+
+	return (
+		<>
+			{editMode && (
+				<div>
+					{cards.map((card, index) => (
+						<button
+							className={styles.cardBtn}
+							key={index}
+							onClick={() => toggleCardSide(index)}
```

Comment:
**issue:** It got really messy - we should split it into smaller components, and not keep everything in one place